packages/renderer/src/lib/featured/FeaturedExtensionDownload.spec.ts

```diff
@@ -59,7 +59,7 @@ test('Expect that the install button is hidden if extension is not installable',
 });
 
 test('Expect that we can see the button and click on the install', async () => {
-  const featuredExtension: FeaturedExtension = {
+  let featuredExtension: FeaturedExtension = {
```

Comment:
```suggestion
  const featuredExtension: FeaturedExtension = {
```

I don't see any write after the first initialization?